src/Surfnet/ServiceProviderDashboard/Application/Console/CleanOldProductionEntitiesCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Console;
```

Comment:
I stronly feel this command should reside in the Infrastructure layer. The actual logic might be part of the application logic. But should be encapsulated in a command handler.

Reply:
I thought I did put it there so thanks for noticing this.

Reply:
:+1: